packages/groq-builder/src/commands/project.test.ts

```diff
@@ -669,7 +669,7 @@ describe("project (object projections)", () => {
           price: true,
         })
       ).toThrowErrorMatchingInlineSnapshot(
-        '"[groq-builder] Because \'validationRequired\' is enabled, every field must have validation (like `q.string()`), but the following fields are missing it: \\"price\\""'
+        `[TypeError: [groq-builder] Because 'validationRequired' is enabled, every field must have validation (like \`q.string()\`), but the following fields are missing it: "price"]`
```

Comment:
It appears that the snapshot changed here to include the ErrorType. At this point it is unclear why. Will investigate.